src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/ActionSidebarContent.tsx

```diff
@@ -28,17 +31,67 @@ import {
 
 const displayName = 'v5.common.ActionsContent.partials.ActionSidebarContent';
 
+const MSG = defineMessages({
+  noReputationErrorTitle: {
+    id: `${displayName}.noReputationErrorTitle`,
+    defaultMessage: 'There is no reputation in this team yet',
+  },
+  noReputationError: {
+    id: `${displayName}.noReputationError`,
+    defaultMessage:
+      'If you have the necessary permissions you can bypass the governance process.',
+  },
+  noPermissionsErrorTitle: {
+    id: `${displayName}.noPermissionsErrorTitle`,
+    defaultMessage: `You don't have the right permissions to create this action type. Choose another action.`,
+  },
+});
+
 const ActionSidebarFormContent: FC<ActionSidebarFormContentProps> = ({
   getFormOptions,
   isMotion,
 }) => {
+  const { formatMessage } = useIntl();
+
+  const { colony } = useColonyContext();
   const { formComponent: FormComponent, selectedAction } =
     useSidebarActionForm();
-  const userHasPermissions = useUserHasPermissions();
-  const form = useFormContext();
-  const { title: titleError } = form.formState.errors;
   const descriptionMetadata = useActionDescriptionMetadata();
 
+  const {
+    setValue,
+    formState: {
+      errors: {
+        title: titleError,
+        domainHasReputation: noReputationError,
+        userHasPermissions: noPermissionsError,
+      },
+    },
+    watch,
+  } = useFormContext();
+  const { actionType, createdIn } = watch();
+  const createdInDomainId = Number(createdIn);
+
+  const userHasPermissions = useUserHasPermissions(
+    actionType,
+    createdInDomainId,
+  );
+  useEffect(() => {
+    setValue('userHasPermissions', userHasPermissions, {
+      shouldValidate: true,
+    });
+  }, [setValue, userHasPermissions]);
+
+  const domainHasReputation = useColonyHasReputation(
+    colony?.colonyAddress,
+    createdInDomainId,
+  );
+  useEffect(() => {
+    setValue('domainHasReputation', domainHasReputation, {
+      shouldValidate: true,
+    });
+  }, [setValue, domainHasReputation]);
```

Comment:
To enable validation on submit button click, the values of the checks now live inside hook-form and are validated using the yup schema.